content/influxdb/v2.0/influxdb-templates/cloud.md

```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
 products: [cloud]
 ---
 
-Use the [`influx` command line interface (CLI)](/influxdb/v2.0/reference/cli/influx/)
-to apply and manage templates in your InfluxDB Cloud account.
+To use templates in InfluxDB Cloud, you have a couple options:
+
+- [Install and customize templates directly in the Cloud UI](#install-and-customize-a-template-in-the-cloud-ui)
+- [Apply and manage templates using the `influx` CLI](#apply-and-manage-templates-using-the-influx-cli)
+
+## Install and customize a template in the Cloud UI
+
+{{% note %}}
+To ensure the template works, complete the **ReadMe** instructions after installing a template. For example, you may need to create a bucket with a specific name in your Cloud instance before the template can be used.
```

Comment:
```suggestion
To ensure the template works, complete the **ReadMe** instructions after installing a template. For example, you may need to configure telegraf to write specific data to your Cloud instance before the template can be used.
```

suggestion: telegraf is by far the most common configuration instruction we ran into when going over the testing document: https://docs.google.com/spreadsheets/d/1-jLiZtxOImQBKLZf5uWAptxfJn_LciBgFmzvChbB1oo/edit#gid=0